### PR TITLE
Subscribe Page Complete

### PR DESCRIPTION
**Pull request by prakritip0**
# Summary of change

* This is a subscription page
* On clicking the submit button, a modal pops, wherein clicking the yes button refreshes the current input

## Type of change

- [x] New feature

# How Has This Been Tested?

- [x] Manual testing

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: images/Homepage.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <link rel="icon" type="image/x-icon" href="file:///Users/kriti/Desktop/GlobalyHub/globaly-task-1/images/favicon.ico">
+    <title>Homepage</title>
+
+</head>
+
+<body>
+    <div class="home">
+        <nav>
+            <a href="file:///Users/kriti/Desktop/GlobalyHub/globaly-task-1/index.html">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokewidth="{1.5}" stroke="currentColor">
```
Comment (bisheshbh) [inline images/Homepage.html]: Use relative url reference here

File: images/Homepage.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                <a href="file:///Users/kriti/Desktop/GlobalyHub/globaly-task-1/index.html" style="height: 4erm;"></a>
+                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" style="height: 100%;width: 100%;">
+                    <path fill-rule="evenodd" d="M7.72 12.53a.75.75 0 010-1.06l7.5-7.5a.75.75 0 111.06 1.06L9.31 12l6.97 6.97a.75.75 0 11-1.06 1.06l-7.5-7.5z" clip-rule="evenodd"></path>
+                </svg>
+                
+            </div>
+            <p>Want more updates on people I meet at Globaly?</p>
+            <h2>Subscribe to my weekly Newsletter</h2>
+            <p>Enter your email:</p>
+            <input type="email" placeholder="yourname@gmail.com">
```
Comment (bisheshbh) [inline images/Homepage.html]: Maybe we can some email validation here ?

File: images/Homepage.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            <br>
+            <button id="submitSubscription">Subscribe</button>
+        </div>
+
+
+    </div>
+
+    <script src="Homepage_files/homepage.js"></script>
```
Comment (bisheshbh) [inline images/Homepage.html]: The folder naming could be something like this : 

```
js/homepage/homepage.js
```

File: images/Homepage_files/homepage.css [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+.pointerDescription {
+    visibility: hidden;
+}
+
+nav svg {
+    color: #B1B9D8;
+    height: 100%;
+    width: 100%;
+    padding: 0.5rem;
+
```
Comment (bisheshbh) [inline images/Homepage_files/homepage.css]: Make sure to create a global css file for common styles.

File: images/Homepage.html (hunk 2 above)
Comment (saras-sth) [inline images/Homepage.html]: name and id attributes are missing in input type

File: images/Homepage.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+            <div class="subscribeHeader">
+                <a href="file:///Users/kriti/Desktop/GlobalyHub/globaly-task-1/index.html" style="height: 4erm;"></a>
+                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" style="height: 100%;width: 100%;">
+                    <path fill-rule="evenodd" d="M7.72 12.53a.75.75 0 010-1.06l7.5-7.5a.75.75 0 111.06 1.06L9.31 12l6.97 6.97a.75.75 0 11-1.06 1.06l-7.5-7.5z" clip-rule="evenodd"></path>
+                </svg>
+                
+            </div>
+            <p>Want more updates on people I meet at Globaly?</p>
+            <h2>Subscribe to my weekly Newsletter</h2>
+            <p>Enter your email:</p>
```
Comment (saras-sth) [inline images/Homepage.html]: have this as a label, and link it with its corresponding input, refer to this link: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

File: images/Homepage.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+            <input type="email" placeholder="yourname@gmail.com">
+            <br>
+            <button id="submitSubscription">Subscribe</button>
```
Comment (saras-sth) [inline images/Homepage.html]: Wrap button and input elements inside a form tag

File: js/subscribe.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
+const modal = document.getElementById("submitModal");
+const submitSubscription = document.getElementById("submitSubscription");
+const congratsText = document.getElementById("congrats");
+const emailInput = document.getElementById("emailAddress");
+
+// function subscriptionButton() {
+//     modal.style.display = "block";
+// }
```
Comment (saras-sth) [inline js/subscribe.js]: remove unnecessary lines

File: js/subscribe.js [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+const modalOptYes = document.getElementById("modalYes");
+const modalOptNo = document.getElementById("modalNo");
+
+
+
+const validateEmail = () => {
+    console.log(emailInput.value)
+    const regx = /^([a-zA-Z0-9\._]+)@([a-zA-Z0-9])+.([a-z]+)(.[a-z]+)?$/
+
+    console.log(regx.test(emailInput.value));
```
Comment (saras-sth) [inline js/subscribe.js]: remove console.log before pushing

File: js/subscribe.js [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+    if (regx.test(emailInput.value)) {
+        submitSubscription.disabled = false;
```
Comment (saras-sth) [inline js/subscribe.js:18]: is the button disabled initially? if not please disable the button initially and only set it to false once the test passes. a nice touch would be to change the button color to a lighter color scheme to indicate disabled state

File: js/subscribe.js [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+// function subscriptionButton() {
+//     modal.style.display = "block";
+// }
+
+const modalOptYes = document.getElementById("modalYes");
+const modalOptNo = document.getElementById("modalNo");
+
+
+
+const validateEmail = () => {
```
Comment (saras-sth) [inline js/subscribe.js:12]: This function is also available in homepage.js. Please D.R.Y. Try to have a common file where both the homepage and subscribe page can access this function

File: images/Homepage_files/homepage.css [hunk 11]
```diff
@@ -0,0 +1,394 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@400;700&family=Outfit:wght@100;400&display=swap');
+
+* {
```
Comment (saras-sth) [inline images/Homepage_files/homepage.css]: move common styles into a different folder such as 'main.css', do not repeat yourself

File: images/Homepage.html (hunk 2 above)
Comment (prakritip0) [reply images/Homepage.html]: I've deleted the Homepage.html file because it was the same as index.html on Homepage branch.

**Pull request merged**